src/test/java/com/truelayer/java/integration/PaymentsProvidersIntegrationTests.java

```diff
@@ -20,19 +23,25 @@ public class PaymentsProvidersIntegrationTests extends IntegrationTests {
     @SneakyThrows
```

Comment:
Can we introduce a unit test on the new PaymentsProviderHandler class that tests that:
- the default scopes are used if not passed at creation time
- the handler calls the get provider endpoint as expected using the IPaymentsProvider collaborator. You can get inspiration from the 2 tests defined [here](https://github.com/TrueLayer/truelayer-java/blob/593478af1350f0852f363bfabfa7773b8cfdebc6/src/test/java/com/truelayer/java/payments/PaymentsHandlerTests.java#L41) for the payments handler